tests/integration/test_storage_s3_queue/test.py

```diff
@@ -822,7 +823,7 @@ def get_count(node, table_name):
 
 def test_max_set_age(started_cluster):
     node = started_cluster.instances["instance"]
-    table_name = f"max_set_age"
+    table_name = f"max_set_age_{uuid4().hex}"
```

Comment:
there is also no other table created with name "max_set_age" as it was before so it was already unique, isn't it?

Reply:
With `--count 10` we run test with same clickhouse and minio. I.e. we need to remove tables we created during a test to allow next run to be successful. 
To simplify work with minio we do not remove anything, but use different paths in each test run.

Reply:
> With --count 10

but we do not use it in CI AFAIK, unless integration tests flaky check may be, but it is run only if the test is changed in the PR